packages/devtools_app/lib/src/standalone_ui/vs_code/debug_sessions.dart

```diff
@@ -139,19 +142,13 @@ class _DevToolsMenu extends StatelessWidget {
         ? TextDirection.rtl
         : TextDirection.ltr;
 
-    Widget devToolsButton(ScreenMetaData screen) {
-      final title = screen.title ?? screen.id;
-      String? disabledReason;
-      if (isRelease) {
-        disabledReason = 'Not available in release mode';
-      } else if (screen.requiresFlutter && !isFlutter) {
-        disabledReason = 'Only available for Flutter applications';
-      } else if (screen.requiresDebugBuild && !isDebug) {
-        disabledReason = 'Only available in debug mode';
-      } else if (screen.requiresDartVm && isWeb) {
-        disabledReason = 'Not available when running on the web';
-      }
-
+    Widget devToolsButton(
```

Comment:
instead of using a helper method, can you create a helper widget like in my draft PR here: https://github.com/flutter/devtools/pull/6709/files#diff-e976c2386d5c7dc8f65bcad658ee5fba12faf0c519c2ad34664c78465fd9129fR275-R306

Reply:
Done! (unfortunately you're going to have some conflicts here)